data/ship builders jobs.txt

```diff
@@ -0,0 +1,132 @@
+			`I only need you to help us travel to the nearby planet Skymoot to deliver sushi for a formal dinner. You will be paid for your expenses.`
+			`Will you help us?"`
+			choice
+				`	"Isn't it dangerous to transport food in an untested prototype?`
+					goto "risky transport"
+				`	"Sure, I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+			label "risky transport"
+			`	"No it's actually more of a formality. Those systems have gone through a lot of simulation cycles already."`
+			choice
+				`	"I'll help you."`
+					accept
+				`	"Sorry, I don't have time to help you."`
+					decline
+	on complete
+		dialog
+			`You are waiting for the Hauler prototype to land, but it appears to stay in orbit.`
+			`There is no response to your hails by radio. Looks like you need to take off and investigate.`
+
+
+mission "Escort Hauler prototype to student party"
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver cooked fish dishes to an archeology students party."
+	source "Skymoot"
+	destination "Zug"
+	landing
+	to offer
+		has "Escort Hauler prototype to formal dinner: offered"	
+	npc accompany
+		government "Merchant"
+		ship "Hauler III" "SB-Labs Cargo Cooker"
+	on enter
+		dialog
+			`As soon as you take off you receive an angry voice message over a short-range inter-ship communications channel: "Why were you trying to contact me over radio! Does everybody here need to know that we are flying around here???`
+			`Let's go to <destination>." You wonder why the destination changed.`
+	on complete
+		dialog
+			`This time the Hauler prototype appears to be landing right behind you, but the ship executes a go-around just before touching the landing pad and flies away at full thrust.`
+			`You receive a short text-message over the short-range inter-ship communiation channel: "The colleagues at the shipyard cannot see us like this!"`
+
+mission "Escort Hauler prototype to factory"
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver overcooked fish to a floating catfood factory."
+	source "Zug"
+	destination "Mere"
+	landing
+	to offer
+		has "Escort Hauler prototype to student party: offered"
+	npc accompany
+		government "Merchant"
+		ship "Hauler III" "SB-Labs Cargo Processor"
+	on complete
+		dialog
+			`The Hauler VI prototype descends at full thrust and lands way ahead of you on a landing pad from a floating catfood factory.`
+			`Your infrared sensors show that the ship emits a lot of heat, but it not from the heavily used thrusters but from the cargo-hold.`
+			`Ms. Oxygenpocket steps out of the ship and says: "This is no good for living creatures, we go to <destination>."`
+			`She enters the prototype and takes off before you have time to respond. Apparently there is something seriously wrong with the cargo that was formerly known as sushi.`
```

Comment:
It's probably better to use "boards" here rather than "enters."
"Prototype" is potentially ambiguous here. I would change it to "prototype Hauler" or "Testament."

"That was formerly known as" can be concisely rewritten in the [active voice](https://en.wikipedia.org/wiki/Active_voice) (which is more effective writing) as, "that you once thought was sushi."

Reply:
Rewritten this a bit more based on your earlier suggestions as well.